tidy3d/components/simulation.py

```diff
@@ -3258,6 +3258,12 @@ def _projection_mnts_2d(cls, val, values):
                         f"Monitor '{monitor.name}' is not supported in 1D simulations."
                     )
 
+                if not monitor.far_field_approx:
+                    raise SetupError(
+                        f"Exact far-field projection for 2D simulations is not yet available for Monitor '{monitor.name}'. "
+                        "Currently, only `far_field_approx = True` is supported."
```

Comment:
We don't use markdown syntax in error messages, so just `'far_field_approx = True'`.